src/edit/selection.js

```diff
@@ -19,9 +23,14 @@ export class SelectionState {
     this.lastNode = null
 
     pm.content.addEventListener("focus", () => this.receivedFocus())
+
+    // Poll for changes to the DOM selection.
```

Comment:
Is there any reason polling is used? Perhaps because there isn't a reliable DOM event we can listen to for knowing that the DOM selection has changed?